nginx/webrick/templates/site.conf.j2

```diff
@@ -7,7 +7,7 @@ upstream {{ site_name }} {
 }
 
 server {
-  server_name {{ site_domain | default('""', true) }};
+  server_name {{ site_domain | default('""', true) }} 127.0.0.1;
```

Comment:
The nginx.org [packages for CentOS](http://nginx.org/packages/centos/7/x86_64/RPMS/) contain some default files in `/etc/nginx/conf.d/`. `default.conf` has the following content (comments stripped):

    server {
        listen       80;
        server_name  localhost;
        location / {
            root   /usr/share/nginx/html;
            index  index.html index.htm;
        }
        error_page   500 502 503 504  /50x.html;
        location = /50x.html {
            root   /usr/share/nginx/html;
        }
    }

This prevents port-forwarded urls like http://127.0.0.1:8081/  to be served as intended. Maybe we'd better remove including `/etc/nginx/conf.d/` but that would obsolete parts of the existing tasks.